docs/source/faq.rst

```diff
@@ -105,17 +107,17 @@ An example implementing this approach is available here :ref:`here <lbl_raw_imag
 
 .. _lbl_imageJ_not_found:
 
-Why isn't ImageJ found by the Show function (RuntimeError: Exception thrown...)?
----------------------------------------------------------------------------------
+Why isn't Fiji or ImageJ found by the Show function (RuntimeError: Exception thrown...)?
+----------------------------------------------------------------------------------------
 
-The SimpleITK ``Show`` function expects the ImageJ program to be installed in
-specific locations. The recommended installation locations are:
+The SimpleITK ``Show`` function expects the Fiji or ImageJ application to be
+installed in specific locations. The recommended installation locations are:
 
 - On Windows: in your user directory (e.g. C:\\Users\\your_user_name\\Fiji.app).
 - On Linux: in ~/bin .
 - On Mac: in /Applications .
```

Comment:
Possibly change recommendation on Mac to ~/Applications so that it doesn't require admin privileges?